ports/cpp-jsonlib/portfile.cmake

```
@@ -0,0 +1,24 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)

```

Is this a Windows limitation (dllexport)? If yes,
```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
endif()

```